bip-musig2/reference.py

```diff
@@ -161,16 +164,16 @@ def cpoint_extended(x: bytes) -> Optional[Point]:
 
 KeyGenContext = namedtuple('KeyGenContext', ['Q', 'gacc', 'tacc'])
 
-def get_pk(keygen_ctx: KeyGenContext) -> bytes:
+def get_pk(keygen_ctx: KeyGenContext) -> XonlyPk:
```

Comment:
Should this be renamed to `get_xonly_pk` or something, to match the spec?

Reply:
done